src/main/java/de/thomas_oster/liblasercut/RasterPart.java

```diff
@@ -38,12 +38,7 @@ public RasterPart(GreyscaleRaster image, LaserProperty laserProperty, Point offs
     this.resolution = resolution;
     this.blackPixelProperty = laserProperty;
     this.whitePixelProperty = blackPixelProperty.clone();
-    if (whitePixelProperty instanceof FloatPowerSpeedFocusFrequencyProperty || whitePixelProperty instanceof FloatPowerSpeedFocusProperty) {
-      whitePixelProperty.setProperty("power", 0.0f);
-    }
-    else {
-      whitePixelProperty.setProperty("power", 0);
-    }
+    whitePixelProperty.setProperty("power", 0f);
```

Comment:
this is a bug (probably due to my changes), was fixed in my PR https://github.com/kkaempf/LibLaserCut/pull/1/files